extension/src/popup/basics/Forms/index.tsx

```diff
@@ -213,3 +209,26 @@ export const RadioField = ({
     {label}
   </CheckAndRadioWrapperEl>
 );
+
+// CSS Form Basic
+
+interface FormRowsProps {
+  children: React.ReactNode;
+}
+export const FormRows = ({ children }: FormRowsProps) => (
+  <div className="FormRows">{children}</div>
+);
+
+interface SubmitButtonWrapperProps {
+  children: React.ReactNode;
+}
+export const SubmitButtonWrapper = ({ children }: SubmitButtonWrapperProps) => (
+  <div className="SubmitButtonWrapper">{children}</div>
+);
+
+interface ErrorMessageProps {
+  children: React.ReactNode;
+}
+export const FormError = ({ children }: ErrorMessageProps) => (
+  <div className="FormError">{children}</div>
+);
```

Comment:
this is the start of adding useful reusable components for the new design. Not perfect yet, but the idea is we'd keep adding css styled components here and slowly deleting the styled components from the old design up above